src/item.h

```diff
@@ -1561,13 +1561,17 @@ class item : public visitable<item>
          * Returns clothing layer for item.
          */
         layer_level get_layer() const;
+
+        /*
+         * Returns the average coverage of each piece of data this item
+         */
+        int get_avg_coverage() const;
         /**
-         * Returns the relative coverage that this item has when worn.
-         * Values range from 0 (not covering anything, or no armor at all) to
-         * 100 (covering the whole body part). Items that cover more are more likely to absorb
-         * damage from attacks.
+         * Returns the highest coverage that any piece of data that this item has that covers the bodypart.
+         * Values range from 0 (not covering anything) to 100 (covering the whole body part).
+         * Items that cover more are more likely to absorb damage from attacks.
          */
-        int get_coverage() const;
+        int get_coverage( const bodypart_id &bodypart = bodypart_id( "num_bp" ) ) const;
```

Comment:
```suggestion
        int get_coverage( const bodypart_id &bodypart ) const;
```

Reply:
This code is here to allow calling get_coverage without a bodypart. Which I agree is a bad thing, but there are a few instances of get_coverage (as well as get_encumber) called in the code that have no current way of knowing the bodypart they are associated with.

I would also like to remove this, but it will require refactoring those. Thats the only reason this is still here.